ow_regolith/include/ServiceClientFacade.h

```diff
@@ -0,0 +1,84 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
```

Comment:
A brief explanation of this class and its motivation would be helpful here.   I may try it in ow_plexil.

Reply:
Class explanation added for both ServiceClientFacade and RegolithSpawner